src/javascript/app/ng-wig/ng-wig.component.spec.js

```diff
@@ -0,0 +1,263 @@
+describe('component: ngWig', () => {
+    let component;
+    let controller;
+    let scope;
+    let content;
+    let options;
+    let buttons;
+    let beforeExecCommand;
+    let afterExecCommand;
+    let element;
+    let attrs;
+    let pasteSpy;
+    let beforeExecSpy;
+    let afterExecSpy;
+    let mocks;
+    let window;
+    let compile;
+
+    beforeEach(module('ngWig'));
+
+    beforeEach(inject((_$componentController_, _$rootScope_, _$window_, _$compile_, _ngWigToolbar_) => {
+        window = _$window_;
```

Comment:
In order to spy on the prompt function of the window object, we need to inject it in the beforeEach statement and hold a reference to it. Otherwise we should inject it in each test that we are spying on it.